tests/assets/scholarly-articles-article1-response.xml

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="UTF-8"?><item xmlns="http://libero.pub" xmlns:jats="http://jats.nlm.nih.gov"><meta><id>article1</id><service>scholarly-articles</service></meta><jats:article><jats:front><jats:article-meta><jats:title-group><jats:article-title>Article 1</jats:article-title></jats:title-group></jats:article-meta></jats:front></jats:article></item>
```

Comment:
Probably best to format (and add a trailing new line). While this is for tests, [hard for humans to read/to diff 1 line XML](https://github.com/elifesciences/elife-article-xml/blob/master/articles/elife-09560-v1.xml)!

Reply:
I chose this format as that how it's received in a response. I'll reformat.